addons/portal/static/src/embed/chatter/thread_service_patch.js

```diff
@@ -0,0 +1,11 @@
+/* @odoo-module */
+
+import { ThreadService } from "@mail/core/common/thread_service";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(ThreadService.prototype, {
+    get ChatterFetchRoute() {
+        return "/mail/chatter_fetch";
+    },
```

Comment:
Again `PascalCase` is reserved for Classes and Types. Variables and props and getters for other things must be camelCase or (SHOUTING_CASE for the louding constants)